en/components/grids_and_lists.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Data Grids and Lists Components
+_description: Ignite UI for Angular provides the fastest Angular Data Grid, high-speed rendering, and Material-based components for a powerful Angular grid and features. 
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Grid, Angular Data Grid, Angular Grid Control, Angular Grid Component
+---
+
+<div class="feature feature--hero">
+  <div class="feature__details">
+    <h2 id="the-fastest-data-angular-data-grid">The Fastest Angular Data Grid</h2>
+    <p>Ignite UI for Angular’s <a href="/components/grid.html">Data Grid</a> is built for optimization, speed, high-performance, and smooth handling of large data sets. With an updated virtualization directive and powerful grid features, your data grid is ready for no-lag scrolling through millions of data points.</p>
+    <a class="cta-btn" href="https://www.google.com">get started for free</a>
+  </div>
+  <div class="feature__image feature__image--right">
+    <img src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid.svg" alt="Ignite UI for Angular's Data Grid optimized for speed and performance.">
+  </div>
+</div>
+<hr />
+<div class="divider"></div>
+
+<section class="feature__container">
+  <div class="feature">
+    <div class="feature__image feature__image--right">
+      <img class="b-lazy b-lazy-gifs" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-768.jpg" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-480.jpg 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-768.jpg 768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-1100.jpg 1100w" data-gif-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-768.gif" data-gif-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-480.gif 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-768.gif 768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-virtualization-performance-1100.gif 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="The virtualization directive brings smooth scrolling to the Angular Data Grid">
```

Comment:
For the URLs pointing to production (`https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png`) we can make them point to different domains, depending on the environment. I'm only mentioning this in case you want to do it. It would be a separate task though. I'll verify this "as is" for the time being.